Resources/public/js/alloyeditor/buttons/embed.jsx

```diff
@@ -51,6 +51,20 @@ YUI.add('ez-alloyeditor-button-embed', function (Y) {
         },
 
         /**
+         * Checks if the command is disabled in the current selection.
+         *
+         * @method isDisabled
+         * @return {Boolean} True if the command is disabled, false otherwise.
+         */
+        isDisabled: function () {
+            var path = this.props.editor.get('nativeEditor').elementPath();
+
+            // http://docs.ckeditor.com/#!/api/CKEDITOR.dom.elementPath
+            // There is also isContextFor( tag ) which potential could have been used.
+            return path && path.contains('table', true) !== null;
```

Comment:
quickly tried to put that in a re-usable API to avoid duplication and here is what can be done:
in Resources/public/js/alloyeditor/plugins/embed.js you can write something like:

```js
CKEDITOR.plugins.add('ezembed', {                                           
    requires: 'widget,ezaddcontent',                                                                                                                                                                                                       
                                                                                                                                                                                                                                               
    init: function (editor) {
        editor.ezembed = {
            canBeAdded: function () { // feel to suggest a better name
                var path = editor.elementPath();

                return path && path.contains('table', true) !== null;
            }
        }

        editor.widgets.add('ezembed', {
        // ...
    },
```

and here then you can write:

```js
_isDisabled: function () {
    return this.props.editor.get('nativeEditor').ezembed.canBeAdded();
}
```
```